supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/MachineConstraints.scala

```diff
@@ -8,10 +8,18 @@ import mouse.all._
 import org.slf4j.Logger
 import wdl4s.parser.MemoryUnit
 import wom.format.MemorySize
+import scala.math.BigDecimal.RoundingMode.CEILING
+
+case class TypeCastMismatch(message: String) extends Exception(message)
```

Comment:
I think that this exception class should have more informative name, something like `NotInDoubleRangeException`.